File: khelmel-java/src/main/java/io/wulfcodes/khelmel/model/bingo/GameState.java
package io.wulfcodes.khelmel.model.bingo;

public enum GameState {
    WAITING,    // < 2 players connected
    LOBBY,      // 2 players connected, not both ready
    COUNTDOWN,  // both ready, countdown in progress
    PLAYING,    // game in progress
    FINISHED    // game over
}

File: khelmel-js/src/models/game.state.js
const GameState = {
    WAITING: 'WAITING',     // < 2 players connected
    LOBBY: 'LOBBY',       // 2 players connected, not both ready
    COUNTDOWN: 'COUNTDOWN',   // both ready, countdown in progress
    PLAYING: 'PLAYING',     // game in progress
    FINISHED: 'FINISHED'     // game over
};

module.exports = GameState;
